apps/backoffice-v2/src/common/components/molecules/ProcessTracker/hooks/useProcessTrackerLogic/process-tracker-adapters/process-tracker.abstract.ts

```diff
@@ -0,0 +1,17 @@
+import { TPlugin } from '@/domains/workflow-definitions/fetchers';
+import { TWorkflowById } from '@/domains/workflows/fetchers';
+
+export interface ProcessTrackerItem {
```

Comment:
If we don't intend on exporting this type from a package let's make sure naming is consistent.
```suggestion
export interface IProcessTrackerItem {
```